features/bewoning_zoeken_met_burgerservicenummer.feature

```diff
@@ -0,0 +1,67 @@
+    | 1961-12-30                 | 1993-02-15 | Javaplein 11     | 345678901                           |
+
+Scenario: Zoeken met een peildatum
+    Als bewoningen wordt gezocht met burgerservicenummer 999994669 en peildatum vandaag
+    Dan wordt de volgende bewoning teruggegeven
+    | naam openbare ruimte | huisnummer | identificatie code nummeraanduiding |
+    | Beethovenlaan        | 23         | 123456789                           |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 123456789 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot |
+    | 999994669           | 1994-05-08                 | -         |
+
+Scenario: Zoeken vanaf een datum
+    Als bewoningen wordt gezocht met burgerservicenummer 999994669 en datumvan 1993-01-01
+    Dan wordt de volgende bewoning teruggegeven
+    | naam openbare ruimte | huisnummer | identificatie code nummeraanduiding |
+    | Beethovenlaan        | 23         | 123456789                           |
+    | Kerkstraat           | 83         | 234567890                           |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 123456789 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot |
+    | 999994669           | 1994-05-08                 | -         |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 234567890 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot  |
+    | 999994669           | 1993-09-10                 | 1994-05-08 |
+
+Scenario: Zoeken tot en met een datum
+    Als bewoningen wordt gezocht met burgerservicenummer 999994669 en datumtotenmet 1994-01-01
+    Dan wordt de volgende bewoning teruggegeven
+    | naam openbare ruimte | huisnummer | identificatie code nummeraanduiding |
+    | Kerkstraat           | 83         | 234567890                           |
+    | Javaplein            | 11         | 345678901                           |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 234567890 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot  |
+    | 999994669           | 1993-09-10                 | 1994-05-08 |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 345678901 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot  |
+    | 999994669           | 1961-12-30                 | 1993-02-15 |
+
+Scenario: Zoeken in een periode
+    Als bewoningen wordt gezocht met burgerservicenummer 999994669, datumvan 1993-01-01 en datumtotenmet 1994-01-01
+    Dan wordt de volgende bewoning teruggegeven
+    | naam openbare ruimte | huisnummer | identificatie code nummeraanduiding |
+    | Kerkstraat           | 83         | 234567890                           |
```

Comment:
Moet het javaplein hier niet bij? En de bijbehorende bewoning met datum adreshouding tot tot 15-02-1993?

Reply:
klopt. Is aangepast.